frontend/src/components/Map/components/FullScreenControl/index.tsx

```diff
@@ -0,0 +1,27 @@
+import L from 'leaflet';
+import 'leaflet.fullscreen';
+import 'leaflet.fullscreen/Control.FullScreen.css';
+import { useEffect } from 'react';
+import { useMap } from 'react-leaflet';
+
+const FullscreenControl = (props: FullscreenOptions) => {
+  const map = useMap();
+
+  useEffect(() => {
+    // @ts-ignore next-line TS does not recognize the extended fullscreen method
+    const control = L.control.fullscreen(props);
```

Comment:
Isn't `L.control.(fullscreen as any)(props)` or something like that better? I'm don't know a lot of TS so not sure about that but it would avoid false positives if an different error than fullscreen not existing comes up.

Reply:
`L.control` has some methods in the Leaflet typescript definition but not `fullscreen` because it's part of the `leaflet.fullscreen` package which extends `L.control`. 

I don't like this ts-ignore but I don't know how to solve this problem.

Reply:
if the tslint settings allow it, something like :
```suggestion
    const control = (L.control as unknown as { fullscreen: Function }).fullscreen(props);
```

or less overkill

```suggestion
    const control = (L.control as any).fullscreen(props);
```

This way if another error on this line shows up (like the `L` import missing or whatever), tslint will alert us.

Reply:
I don't know enought TS to know if there's a way to inline extending `L.control` to add a `fullscreen` function and adding a whole type for that seems overkill, so `as any` seems the best option